client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/navigator-header/style.scss

```diff
@@ -1,15 +1,18 @@
 @import "@automattic/typography/styles/fonts";
 
 .pattern-assembler {
-	.navigator-header__title {
-		font-family: Recoleta, sans-serif;
-		font-style: normal;
-		font-weight: 400;
-		font-size: $font-title-large;
-		line-height: 40px;
-		letter-spacing: -0.32px;
-		color: var(--color-neutral-100);
+	.navigator-header__heading {
```

Comment:
I found that before and I assumed it was a visual effect since I checked it as followed:

![image](https://user-images.githubusercontent.com/13596067/223391239-794ffbd7-f06f-4274-83ab-d0ea6ba203b4.png)

I didn't notice we have `top: 6px` there. I'll make a change as you suggested!